docs/how-we-hire.md

```diff
@@ -0,0 +1,41 @@
+# How We Hire
+
+We strive to be transparent about our hiring process. While specifics may differ
+from posting to posting, this is our general process.
+
+0. We lint all of our job postings with [joblint.org](https://joblint.org). We
+   always provide a salary range in our postings so there are no surprises for
+   anyone at any stage of the process.
+1. After posting a job, we'll use social media to spread it amongst our network.
+   We're also actively identifying ways to spread it beyond our network in an
+   attempt to find candidates that wouldn't normally think about applying to
+   Stratasan. We normally don't work with recruiters as we don't see the value.
+2. Based on the rate of responses, we'll keep this posting up for 2-4 weeks
+   though perhaps longer.
+3. During this time, we'll conduct phone interviews with candidates meeting the
+   minimal qualifications. This phone call is a behavioral interview and we'll
+   ask questions that don't have right or wrong answers but are designed to get
+   at the candidate's opinions and previous experience. Example questions might
+   be "Describe your ideal deployment pipeline" or "Tell us about a time a
+   release you were involved with didn't go to plan".
+4. Based on these conversations, we'll invite a few candidates to participate in
+   a project. For developer roles, we'll take an existing issue from our issue
+   tracker, build a development environment and let the candidate tackle the
+   issue. This serves to let us see how they approach problems and gives the
+   candidates the opportunity to see the quality of our codebase and how we
+   write issues. For more operations-based roles, the project will involve
+   designing a new service we'd like to run (as it's more difficult to safely
+   carve out a section of our infrastructure to do work). In either case, we'll
+   absolutely compensate you for this time. We don't believe in asking people to
+   do work for free. If a candidate has time constraints outside of normal
+   business hours, we'll work with them to find a good time to complete the
+   work.
+5. Then we'll build a schedule lasting no more than half a day to 1) discuss
+   your work, 2) pair-program with a team member, 3) meet more of the team and
+   4) meet our CEO. We will do this both for candidates local to Nashville and
+      remote candidates.
```

Comment:
Should we add something in here about how remote candidates can be interviewed remotely? My thinking it just so no one is expecting to get flown in, since some companies do that.

Also, maybe something about which tools we can use to do remote interviews and pair programming, since companies vary on this aspect a lot too?